### ISSUE TITLE
not last version reported

### ISSUE THREAD

**Issue by jesuslop** [pymupdf/PyMuPDF#3616]
### Description of the bug

pymupdf.pymupdf_version evals to 1.24.5 instad of 1.24.6

### How to reproduce the bug

```
C:\Users\xxx>pip show pymupdf
Name: PyMuPDF
Version: 1.24.6
Summary: A high performance Python library for data extraction, analysis, conversion & manipulation of PDF (and other) documents.
Home-page:
...

C:\Users\xxx>python
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pymupdf
>>> pymupdf.pymupdf_version
'1.24.5'
```

### PyMuPDF version

1.24.6

### Operating system

Windows

### Python version

3.10

**Comment by julian-smith-artifex-com** [comment]
You're quite right, thanks for letting us know.

[The version/date-checking code went wrong because of the rename of `VersionBind` => `pymupdf_version`.]

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.24.7.
